VirtoCommerce.Storefront.Tests/Model/MoneyOperationTests.cs

```diff
@@ -0,0 +1,40 @@
+using VirtoCommerce.Storefront.Model.Common;
+using Xunit;
+using StorefrontLanguage = VirtoCommerce.Storefront.Model.Language;
+
+namespace VirtoCommerce.Storefront.Tests.Model
+{
+    public class MoneyOperationTests
```

Comment:
Could you add Traits("CI")  attribute for test class?

Reply:
Sorry, I forgot about that attribute. Added it.